test/functional/maxuploadtarget.py

```diff
@@ -54,7 +54,7 @@ def run_test(self):
         # p2p_conns[2] will test resetting the counters
         p2p_conns = []
 
-        for i in range(3):
+        for _ in range(3):
```

Comment:
Interested in the use of the underscore as the variable name here, did some reading and from what I've read use of the underscore only works as something other than a variable name when used in an interactive python shell? Other than that people seem to believe it's a bad naming convention.

Context: https://stackoverflow.com/questions/26895362/what-does-in-python-do

Reply:
`_` as a variable name in python is a convention to indicate that the result won't be used.

Reply:
Thanks for the explanation, makes sense.